payments-core/src/main/java/com/stripe/android/payments/core/injection/Stripe3DSAuthenticatorModule.kt

```diff
@@ -31,28 +28,31 @@ internal class Stripe3DSAuthenticatorModule {
     @IntoMap
     @IntentAuthenticatorKey(NextActionData.SdkData.Use3DS2::class)
     internal fun provideStripe3DSAuthenticator(
-        context: Context,
         @Named(ENABLE_LOGGING) enableLogging: Boolean,
-        stripeRepository: StripeRepository,
-        webIntentAuthenticator: WebIntentAuthenticator,
-        paymentRelayStarterFactory: (AuthActivityStarterHost) -> PaymentRelayStarter,
-        analyticsRequestExecutor: AnalyticsRequestExecutor,
-        analyticsRequestFactory: AnalyticsRequestFactory,
-        @IOContext workContext: CoroutineContext,
-        @UIContext uiContext: CoroutineContext
+        threeDs1IntentReturnUrlMap: MutableMap<String, String>
     ): PaymentAuthenticator<StripeIntent> {
         return Stripe3DS2Authenticator(
             PaymentAuthConfig.get(),
+            enableLogging,
+            threeDs1IntentReturnUrlMap
+        )
+    }
+
+    @Provides
+    internal fun provideStripe3ds2ChallengeResultProcessor(
```

Comment:
as commented in the other thread, this can be removed for now

Reply:
Sorry is this removed? still see it being injected. This will not create any object as it's not used by dagger anywhere